test/functional/system/common.js

```diff
@@ -177,7 +180,12 @@ function addTransactionToUnconfirmedQueue(library, transaction, cb) {
 	);
 }
 
-function addTransactionsAndForge(library, transactions, cb) {
+function addTransactionsAndForge(library, transactions, forgeDelay, cb) {
+	if (typeof forgeDelay === 'function') {
+		cb = forgeDelay;
+		forgeDelay = 800;
```

Comment:
Why we need to support that delay here?

Reply:
@4miners As per @SargeKhan it was added because of transaction pool. He can explain further on it.